testsuite/tests/ui/conftest.py

```diff
@@ -379,6 +387,48 @@ def get_resultsdir_path(node):
     return path
 
 
+def fullpage_screenshot(driver, file_path):
+    """
+        A full-page screenshot function. It scroll the website and screenshots it.
+        - Creates multiple files
+        - Screenshots are made only vertically (on Y axis)
+    """
+    if driver is None:
```

Comment:
Is this necessary?  This should handle try block

Reply:
removed (I added this before the try-block to avoid a crash)